i18n/translation/translation.go

```diff
@@ -44,27 +45,39 @@ func NewTranslation(data map[string]interface{}) (Translation, error) {
 			return nil, err
 		}
 		return &singleTranslation{id, tmpl}, nil
-	case map[string]interface{}:
-		templates := make(map[language.Plural]*template, len(translation))
+	case map[interface{}]interface{}:
+		// The YAML parser uses interface{} keys.
```

Comment:
Can you make this comment more detailed?
"The YAML parser uses interface{} keys so first we convert them to string keys."